apps/builder/app/builder/shared/code-editor.tsx

```diff
@@ -25,6 +25,20 @@ import { CrossIcon, MaximizeIcon, MinimizeIcon } from "@webstudio-is/icons";
 
 const ExternalChange = Annotation.define<boolean>();
 
+const minHeightVar = "--ws-code-editor-min-height";
+const maxHeightVar = "--ws-code-editor-max-height";
+
+export const getMinMaxHeightVars = ({
```

Comment:
this was our guideline all along - typeed interface when we set css variables across modules
this way exactvariables and values are implementation detail and we get a typed interface